charts/greptimedb-standalone/values.yaml

```diff
@@ -0,0 +1,167 @@
+image:
+  registry: docker.io
+  # The image repository
+  repository: greptime/greptimedb
+  # The image tag
+  tag: "v0.4.0"
+  # The image pull policy for the controller
+  pullPolicy: IfNotPresent
+  # The image pull secrets.
+  pullSecrets: []
+
+# Overrides the chart's name
+nameOverride: ""
+
+# Provide a name to substitute for the full names of resources
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+# Command to execute instead of defined in Docker image
+command: [
+  "greptime", "standalone", "start"
```

Comment:
In standalone mode, the start command is always `greptime standalone start <args>`. We should keep `args` configurable instead of the whole command.